specification/securitycenter/resource-manager/Microsoft.Security/preview/2017-08-01-preview/security.json

```diff
@@ -0,0 +1,1288 @@
+                    "x-ms-enum": {
+                        "name": "provisioningSetting",
+                        "modelAsString": false
+                    }
+                }
+            },
+            "required": [
+                "provisioningSetting"
+            ]
+        },
+        "ComplianceList": {
+            "type": "object",
+            "readOnly":true,
+            "description": "List of Compliance objects",
+            "properties": {
+                "value": {
+                    "readOnly": true,
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Compliance"
+                    }
+                },
+                "nextLink": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The URI to fetch the next page."
+                }
+            }
+        },
+        "Compliance": {
+            "type": "object",
+            "readOnly":true,
+            "description": "Compliance of a scope",
+            "properties": {
+                "properties": {
+                    "$ref":"#/definitions/ComplianceProperties"
+                }
+            },
+            "allOf": [{
+                "$ref": "#/definitions/Resource"
+            }]
+        },
+        "ComplianceProperties": {
+            "readOnly": true,
+            "type": "object",
+            "description": "The Compliance score (percentage) of a Subscription is a sum of all Resources' Compliances under the given Subscription. A Resource Compliance is defined as the compliant ('healthy') Policy Definitions out of all Policy Definitions applicable to a given resource.",
+            "properties": {
+                "assessmentTimestampUtc": {
+                    "readOnly": true,
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "The timestamp when the Compliance calculation was conducted."
+                },
+                "resourceCount": {
+                    "readOnly": true,
+                    "type": "integer",
+                    "description": "The resource count of the given subscription for which the Compliance calculation was conducted (needed for Management Group Compliance calculation)."
+                },
+                "assessmentResult": {
+                    "readOnly": true,
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ComplianceSegment"
+                    },
+                    "description": "An array of segment, which is the actually the compliance assessment."
+                }
+            }
+        },
+        "ComplianceSegment": {
+            "readOnly": true,
+            "type": "object",
+            "description": "A segment of a compliance assessment.",
+            "properties": {
+                "SegmentType": {

```

Comment:
Are these actually in PascalCase on the wire? General convention is to have properties in camelCase in the json object. The primarty purpose of the swagger spec is to describe the REST API correctly and completely as it happens to be on the wire. Can you please check these models via Fiddleror another network monitoring tool and confirm that this is how they are represented on the wire? Any mistake in the casing will translate to serialization/deserialization bugs in the sdk. Unknown properties will be dropped during [de]serialization.

Reply:
they are camel case, changed the swagger to match